setup/src/Magento/Setup/Test/Unit/Console/Command/DiCompileCommandTest.php

```diff
@@ -70,8 +75,8 @@ public function setUp()
             ->getMock();
         $this->componentRegistrarMock = $this->createMock(\Magento\Framework\Component\ComponentRegistrar::class);
         $this->componentRegistrarMock->expects($this->any())->method('getPaths')->willReturnMap([
-            [ComponentRegistrar::MODULE, ['/path/to/module/one', '/path/to/module/two']],
-            [ComponentRegistrar::LIBRARY, ['/path/to/library/one', '/path/to/library/two']],
+            [ComponentRegistrar::MODULE, ['/path/to/module/one', '/path (1)/to/module/two']],
```

Comment:
Can new values be added, not replaced old ones? As result we will be sure that old and new values works good.

Reply:
Hi @ihor-sviziev, 
Thank you for feedback.

You'd like to have:
```
[ComponentRegistrar::MODULE, ['/path/to/module/one', '/path/to/module/two', '/path (1)/to/module/three']],
[ComponentRegistrar::LIBRARY, ['/path/to/library/one', '/path/to/library/two', '/path (1)/to/library/three']],
```

instead:
```
[ComponentRegistrar::MODULE, ['/path/to/module/one', '/path (1)/to/module/two'],
[ComponentRegistrar::LIBRARY, ['/path/to/library/one', '/path (1)/to/library/two']],
```

Reply:
Ah, I thought there is only one type of path, so now it's ok. Sorry for confusing you